Rework secret-handshake exercise docs (#6)
kytrinyx/problem-specifications: exercises/secret-handshake/instructions.md

# Instructions

Your task is to converts a number between 1 and 31 to a sequence of actions in the secret handshake.

> The sequence of actions are chosen by looking at each of the digits in the binary number.

Reviewer: ```suggestion
The sequence of actions are chosen by looking at each digit of the number once it's been converted to binary.
```

Start at the right-most digit and move left.

The actions for each number place are:

```
00001 = wink
00010 = double blink
00100 = close your eyes
01000 = jump
10000 = Reverse the order of the operations in the secret handshake.
```

> Let's use the number 9 as an example.
> 9 in binary is `1001`.
> 
> The digit that is farthest to the right is 1, so the first action is `wink`.
> Going left, the next digit is 0, so there is no double-blink.
> Going left again, the next digit is 0, so you leave your eyes open.
> Going left again, the next digit is 1, so you jump.
> That was the last digit, so the final code is:

Reviewer: ```suggestion
Let's use the number `9` as an example:

- 9 in binary is `1001`.
- The digit that is farthest to the right is 1, so the first action is `wink`.
- Going left, the next digit is 0, so there is no double-blink.
- Going left again, the next digit is 0, so you leave your eyes open.
- Going left again, the next digit is 1, so you jump.

That was the last digit, so the final code is:
```

```
wink, jump
```

> Given the number 26, which is 11010 in binary, we get the following actions:

Reviewer 2: In the above example, `1001` is put in backticks. Do we want that here too?

Author: Yes, good call.

# Instructions

Your task is to converts a number between 1 and 31 to a sequence of actions in the secret handshake.

> The sequence of actions are chosen by looking at each of the number places in the binary number.
> Starting at the back, looking at the digit that is farthest to the right.
> Then, move left.

Reviewer: Reference to "number place" and "back" confused me.

```suggestion
The sequence of actions are chosen by looking at each of the digits in the binary number.
Start at the right-most digit and move left.
```

Author: Cool. I was trying to figure out how to avoid "least significant digit" 😅

# Instructions

> Your task is to converts a number between 1 and 31 to a sequence of actions in the secret handshake.

Reviewer 2: ```suggestion
Your task is to convert a number between 1 and 31 to a sequence of actions in the secret handshake.
```

> The sequence of actions are chosen by looking at each digit of the number once it's been converted to binary.

Reviewer 2: Maybe be explicit about which digits to check?

```suggestion
The sequence of actions are chosen by looking at the rightmost five digits of the number once it's been converted to binary.
```

Author: The largest number in the range is 5 digits long in binary, so we never have more digits than that.

Reviewer 2: True, but I could imagine programmers thinking about numbers in terms of bytes and multiples of 8 bits.

Author: Yes, that's a good point.

Author: I'm going to add this, and then maybe someone in the problem-specs repo will have some other good idea.
